src/app/(auth)/sign-in/page.jsx

```diff
@@ -1,9 +1,83 @@
-import React from 'react'
+import SignInForm  from "@/components/SignInForm";
+import Image from "next/image";
+import Link from "next/link";
+import React from "react";
+import { FaArrowRightLong } from "react-icons/fa6";
+import { FaArrowLeft } from "react-icons/fa6";
 
 const SignUp = () => {
   return (
-    <div>Sign In</div>
-  )
-}
+    <>
+      <div className="flex">
+        <div className=" hidden md:flex h-screen w-[50%] bg-tbhgreen/80">
+          <iframe
```

Comment:
Fix the width of the video

Reply:
I will work on the video during the next sprint